cobbler.spec

```diff
@@ -384,6 +384,40 @@ chgrp %{apache_group} %{_sysconfdir}/cobbler/settings.yaml
 %else
 %post
 %systemd_post cobblerd.service
+
+if [ $1 -gt 1 ]; then
```

Comment:
What does this mean? I assume this is some specfile magic; perhaps a comment might be useful to explain when this fires?

Reply:
This is just the way to detect a package upgrade. See https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_syntax

I'll add a note anyway!